ports/secretteagarden/README.md

```diff
@@ -0,0 +1,18 @@
+## Notes
+
+Big thanks to [Angus Beer] (https//:angusbeer.itch.io) for this adorable yet challenging game. Also, special thanks to all who tested this port.
```

Comment:
Bad link in the README.md. No space between the close square bracket and open parenthesis. `] (` :)

Reply:
I thought something was fishy about that